Clicker/Extensions/UIColor+Shared.swift

```diff
@@ -9,31 +9,28 @@ import UIKit
-    @nonobjc static let clickerGradientGrey = UIColor(red: 67/255, green: 67/255, blue: 67/255, alpha: 0.0)
-    @nonobjc static let clickerGreen = UIColor(red: 41/255, green: 192/255, blue: 158/255, alpha: 1.0)
-    @nonobjc static let clickerHalfGreen = UIColor(red: 41/255, green: 192/255, blue: 158/255, alpha: 0.5)
-    @nonobjc static let clickerLabelGrey = UIColor(red: 61/255, green: 61/255, blue: 61/255, alpha: 1.0)
-    @nonobjc static let clickerMint = UIColor(red: 6/255, green: 184/255, blue: 158/255, alpha: 0.19)
     @nonobjc static let clickerOrange = UIColor(red: 243/255, green: 132/255, blue: 82/255, alpha: 1.0)
-    @nonobjc static let clickerOptionGrey = UIColor(red: 240/255, green: 240/255, blue: 240/255, alpha: 1.0)
-    @nonobjc static let clickerNavBarGrey = UIColor(red: 246/255, green: 246/255, blue: 248/255, alpha: 1.0)
-    @nonobjc static let clickerNavBarLightGrey = UIColor(red: 249/255, green: 249/255, blue: 249/255, alpha: 1.0)
     @nonobjc static let clickerRed = UIColor(red: 249/255, green: 93/255, blue: 93/255, alpha: 1.0)
-    @nonobjc static let clickerTransparentGrey = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 0.6)
     @nonobjc static let clickerWhite = UIColor(red: 250/255, green: 250/255, blue: 250/255, alpha: 1.0)
-
-    
+    @nonobjc static let clickerGreen0 = UIColor(red: 41/255, green: 192/255, blue: 158/255, alpha: 1.0)
+    @nonobjc static let clickerGreen1 = UIColor(red: 41/255, green: 192/255, blue: 158/255, alpha: 0.5)
+    @nonobjc static let clickerGreen2 = UIColor(red: 6/255, green: 184/255, blue: 158/255, alpha: 0.19)
+    @nonobjc static let clickerGrey0 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 1.0)
+    @nonobjc static let clickerGrey1 = UIColor(red: 229/255, green: 231/255, blue: 237/255, alpha: 1.0)
+    @nonobjc static let clickerGrey2 = UIColor(red: 158/255, green: 167/255, blue: 179/255, alpha: 1.0)
+    @nonobjc static let clickerGrey3 = UIColor(red: 122/255, green: 129/255, blue: 139/255, alpha: 1.0)
+    @nonobjc static let clickerGrey4 = UIColor(red: 247/255, green: 249/255, blue: 250/255, alpha: 1.0)
+    @nonobjc static let clickerGrey5 = UIColor(red: 225/255, green: 225/255, blue: 225/255, alpha: 1.0)
+    @nonobjc static let clickerGrey6 = UIColor(red: 240/255, green: 240/255, blue: 240/255, alpha: 1.0)
+    @nonobjc static let clickerGrey7 = UIColor(red: 246/255, green: 246/255, blue: 248/255, alpha: 1.0)
+    @nonobjc static let clickerGrey8 = UIColor(red: 249/255, green: 249/255, blue: 249/255, alpha: 1.0)
+    @nonobjc static let clickerGrey9 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 0.6)
+    @nonobjc static let clickerGrey10 = UIColor(red: 61/255, green: 61/255, blue: 61/255, alpha: 1.0)
+    @nonobjc static let clickerBlack0 = UIColor(red: 31/255, green: 44/255, blue: 56/255, alpha: 1.0)
+    @nonobjc static let clickerBlack1 = UIColor(red: 32/255, green: 32/255, blue: 32/255, alpha: 1.0)
```

Comment:
not sure how I feel about the color names lol (don't hate me). Lets talk about what you think. Why do you think this is a better approach?

Reply:
The reason I changed this was because before what we would have are names like clickerLightGrey, clickerMediumGrey, clickerDarkGrey, etc. The two problems I had with this is: - we would then have 2 other grey colors, with no real good way left to name them
- the idea behind naming these colors `clickerMediumGrey` is so that we know which grey to use when I look at the design but in reality, I always just go back to this file and compare the RGB values b/c I don't remember which type of grey it was.
So I thought that just naming it by number would solve the issue of coming up with a descriptive name.

Reply:
LMK what you think about ^. curious if theres a better way

Reply:
I agree that this color naming convention isn't any worse than the previous convention including `clickerMediumGrey`, because during development I also always check the RGB values on the design to make sure to use the correct color. Additionally this new convention allows an arbitrary number of colors to be defined with consistent names, a positive feature.